sebastian-correa/final-project/src/components/Home/Home.js

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { AUTH_CODE, BASE_URL, CLIENT_ID, CLIENT_SECRET } from '../Login/LoginHandler';
+import Navbar from '../Navbar/Navbar';
+import Player from '../Player/Player';
+import Visualizer from '../Visualizer/Visualizer';
+import './Home.css';
+
+const queryString = require('query-string');
+
+const Home = () => {
+  const [searchResults, setSearchResults] = useState([]);
+  const [token, setToken] = useState({});
+  const search = useSelector((state) => state.search);
+
+  const requestToken = async () => {
+    return axios('https://accounts.spotify.com/api/token', {
```

Comment:
@Secorvil , even this is not bad, it's a good practice to keep a different code layer for logic and API calls, please research about "thunk", "side effects in react" or "service layer in react".